build_openroad.sh

```diff
@@ -365,12 +366,18 @@ if [ ! -z "${CLEAN_FORCE+x}" ]; then
 else
         CLEAN_CMD="-x -d --interactive"
 fi
+
 if [ ! -z "${CLEAN_BEFORE+x}" ]; then
         echo "[INFO FLW-0016] Cleaning up previous binaries and build files."
         git clean ${CLEAN_CMD} tools
         git submodule foreach --recursive git clean ${CLEAN_CMD}
+        rm -rf "${buildDir}"
```

Comment:
I don't think we need to delete the build dir. The git clean command should control that.